unturned/OpenMod.Unturned/Players/UnturnedPlayer.cs

```diff
@@ -74,6 +78,14 @@ public override int GetHashCode()
 
         public double Health => Player.life.health;
 
+        public IPAddress IP
+        {
+            get
+            {
+                return SteamPlayer.transportConnection.GetAddress();
```

Comment:
Make the IP property nullable because `SteamPlayer.transportConnection` can be null and `.GetAddress()` may return null. Also better use `SteamPlayer.getAddress()`?

Reply:
I agree, also some games may not support it at all